packages/volto/src/components/theme/Search/Search.jsx

```diff
@@ -282,7 +285,7 @@ class Search extends Component {
                   <Pagination
                     activePage={this.state.currentPage}
                     totalPages={Math.ceil(
-                      this.props.search.items_total / settings.defaultPageSize,
+                      this.props.search.items_total / this.defaultPageSize,
```

Comment:
I think here options should be taken into account.
```
const options = qs.parse(this.props.history.location.search);
…

                      this.props.search.items_total / options.b_size ||
                        this.defaultPageSize,
```

Reply:
Made the required changes.